runtime/container.go

```diff
@@ -193,6 +193,18 @@ func Load(root, id, shimName string, timeout time.Duration) (Container, error) {
 		}
 		c.processes[pid] = p
 	}
+
+	_, err = os.Stat(c.bundle)
+	if err != nil && !os.IsExist(err) {
+		for _, p := range c.processes {
```

Comment:
calling `runc delete <containerd-id>` should be enough here, if that fails, that loop can be used as a fallback